whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestBasicAuthTestIntegration.java

```diff
@@ -221,6 +223,19 @@ public void create_failed_with_basic_auth_no_https() {
         assertThat(response.readEntity(String.class), containsString("HTTPS required for Basic authorization"));
     }
 
+    @Test
+    @Disabled("PUT and DELETE fail when using sendError()")
```

Comment:
If we want to keep using sendError as this tests does
we need to tackle when the METHOD is PUT or DELETE

In these situations the error that we are sending is "" even if we do httpResponse.sendError(HttpStatus.UPGRADE_REQUIRED_426, "HTTPS required for Basic authorization") in the code